packages/validators/test/__fixtures__/v2023_09_28/experience.ts

```diff
@@ -244,6 +246,7 @@ export const experience = {
               },
               {
                 definitionId: 'button',
+                displayName: 'Button 1',
```

Comment:
Can we have at least one element without `displayName` in this fixture?

```suggestion
                // Component node without optional `displayName`.
```